### PR TITLE
[terra-signature] Implementation page doc updates

### PR DESCRIPTION
**Pull request by KV106606Viswanath**
### Summary
Updated implementation page documentation regarding consumers' responsibility to ensure the accessibility of the Terra Signature control.

**What was changed:**

**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9215

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 1]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
 import Signature from 'terra-signature';
 ```
 
+## Consuming teams are responsible for
+- **Providing Descriptive Labels**: Ensure that the signature component is labeled clearly and descriptively for screen reader users. This includes providing labels for form fields and instructions for signing.
+
+- **Implementing Keyboard Accessibility**: Ensure that all functionality of the signature component can be accessed and operated using a keyboard alone, without relying on mouse interactions.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:41]:
```suggestion
- **Implementing Keyboard Accessibility**: Ensure that all functionality of terra-signature can be accessed and operated using only a keyboard, without any mouse interactions.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 2]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
...
+
+- **Using ARIA Attributes**: Use ARIA (Accessible Rich Internet Applications) attributes to enhance the accessibility of the signature component, such as aria-label and aria-describedby, to provide additional context for screen reader users.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:43]:
```suggestion
- **Using ARIA Attributes**: Use ARIA (Accessible Rich Internet Applications) [attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) to enhance the accessibility of terra-signature, such as [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) and [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby), to provide additional context for screen reader users.
```

File: packages/terra-core-docs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added implementation page documentation regarding consumers responsibility to ensure the accessibility of the `terra-signature`.
```
Comment (sdadn) [inline packages/terra-core-docs/CHANGELOG.md:6]:
```suggestion
  * Added implementation page documentation regarding consumers responsibility to ensure the accessibility for `terra-signature`.  
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 4]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
 import Signature from 'terra-signature';
 ```
 
+## Consuming teams are responsible for
+- **Providing Descriptive Labels**: Ensure that the signature component is labeled clearly and descriptively for screen reader users. This includes providing labels for form fields and instructions for signing.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:39]:
```suggestion
- **Providing Descriptive Labels**: Ensure that the terra-signature component is labeled clearly and descriptively for screen reader users. This includes providing labels for form fields and instructions for signing.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 5]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
...
+
+- **Implementing Keyboard Accessibility**: Ensure that all functionality of the signature component can be accessed and operated using a keyboard alone, without relying on mouse interactions.
+
+- **Using ARIA Attributes**: Use ARIA (Accessible Rich Internet Applications) attributes to enhance the accessibility of the signature component, such as aria-label and aria-describedby, to provide additional context for screen reader users.
+
+- **Testing with Assistive Technologies**: Test the signature component with a variety of assistive technologies, such as screen readers, to ensure that it is fully accessible and usable by all users.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:45]:
```suggestion
- **Testing with Assistive Technologies**: Test terra-signature with a variety of assistive technologies, such as screen readers, to ensure that it is fully accessible and usable by all users.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 6]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
...
+
+- **Providing Alternative Text for Images**: If the signature component includes any images or icons, provide alternative text that describes the content or function of the image for users who cannot see it.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:47]:
```suggestion
- **Providing Alternative Text for Images**: If terra-signature includes any images or icons, provide alternative text that describes the content or function of the image for users who cannot see it.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 7]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
...
+
+- **Ensuring Sufficient Color Contrast**: Ensure that there is sufficient color contrast between text and background colors to make the signature component readable for users with low vision or color blindness.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:49]:
```suggestion
- **Ensuring Sufficient Color Contrast**: Ensure that there is sufficient color contrast between text and background colors to make terra-signature readable for users with low vision or color blindness.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx [hunk 8]
```diff
@@ -35,6 +35,23 @@ This component requires the following peer dependencies be installed in your app
...
+
+- **Documenting Accessibility Features**: Document the accessibility features of the signature component for end users, including instructions on how to use it with assistive technologies.
```
Comment (sdadn) [inline packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx:51]:
```suggestion
- **Documenting Accessibility Features**: Document the accessibility features of terra-signature for end users, including instructions on how to use it with assistive technologies.
```

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 1 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 2 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/CHANGELOG.md (hunk 3 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/CHANGELOG.md]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 4 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 5 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 6 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 7 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

File: packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx (hunk 8 above)
Comment (KV106606Viswanath) [reply packages/terra-core-docs/src/terra-dev-site/doc/signature/About.1.doc.mdx]: Updated

**Review by sdadn — APPROVED** [review]

**Review by sugan2416 — APPROVED** [review]

**Pull request merged**